src/SFA.DAS.EmploymentCheck.AcceptanceTests/Steps/ParallelEmploymentChecksStepDefinitions.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using AutoFixture;
+using Dapper.Contrib.Extensions;
+using FluentAssertions;
+using HMRC.ESFA.Levy.Api.Types;
+using Microsoft.Data.SqlClient;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EmploymentCheck.Data.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using WireMock.Matchers;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+
+namespace SFA.DAS.EmploymentCheck.AcceptanceTests.Steps
+{
+    [Binding]
+    [Scope(Feature = "EmploymentChecksArePerformedInParallel")]
+    public class ParallelEmploymentChecksStepDefinitions
+    {
+        private readonly TestContext _context;
+        private readonly List<Data.Models.EmploymentCheck> _checks;
+        private List<LearnerNiNumber> _dcApiResponse;
+        private ResourceList _accountsApiResponse;
+
+        public ParallelEmploymentChecksStepDefinitions(TestContext context)
+        {
+            _context = context;
+            _checks = new List<Data.Models.EmploymentCheck>();
+        }
+        
+        [Given(@"a number of unprocessed employment checks")]
+        public async Task GivenANumberOfUnprocessedEmploymentChecks()
+        {
+            await using var dbConnection = new SqlConnection(_context.SqlDatabase.DatabaseInfo.ConnectionString);
+            for (var i = 0; i < 10; i++)
+            {
+                var check = _context.Fixture.Build<Data.Models.EmploymentCheck>()
+                    .Without(c => c.RequestCompletionStatus)
+                    .Without(c => c.Employed)
+                    .Without(c => c.LastUpdatedOn)
+                    .Without(c => c.ErrorType)
+                    .With(c => c.CreatedOn, DateTime.UtcNow.AddMinutes(i-5))
+                    .Create();
+
+                await dbConnection.InsertAsync(check);
+                _checks.Add(check);
+            }
+        }
+
+        //TODO: convert this to DB inserts as this is taking long time for not a lot of gain
```

Comment:
Can we create a tech debt ticket for the TODO items in these steps?

Reply:
created a teck debt ticket https://skillsfundingagency.atlassian.net/browse/EI-1946

Reply:
also fixed the code smell